docker-compose.test.yml

```diff
@@ -0,0 +1,86 @@
+version: '3'
+services:
+  db:
+    image: postgres:9.6
+    environment:
+      POSTGRES_USER: eregs
+      POSTGRES_PASSWORD: sgere
+      POSTGRES_DB: eregs
+    ports:
+      - 5432:5432
+  regulations-parser:
+    build:
+      context: ./regulations-parser
+      dockerfile: Dockerfile.local
+    volumes:
+      - ./regulations-parser:/app/src
+      - ./output:/app/output
+      - eregs-cache:/app/cache
+    environment:
+      PYTHONUNBUFFERED: "1"
+      DATABASE_URL: sqlite:////app/cache/db.sqlite
+      EREGS_CACHE_DIR: /app/cache
+      EREGS_OUTPUT_DIR: /app/output
+    working_dir: /app/src
+    command: "tail -f /dev/null"
+    links:
+      - regulations-core
+  regulations-core:
+    build:
+      context: ${CORE_CONTEXT:-./regulations-core}
+      dockerfile: ${PWD}/config/regulations-core/Dockerfile
+    volumes:
+      - ${CORE_CONTEXT:-./regulations-core}:/app/src
+      - ./config/regulations-core/local_settings.py:/var/lib/eregs/local_settings.py
+    environment:
+      ALLOWED_HOST: regulations-core
+      DJANGO_SETTINGS_MODULE: regcore.settings.pgsql
+      PYTHONPATH: /var/lib/eregs
+    ports:
+      - 8080:8080
+    links:
+      - db
+  regulations-site:
+    build:
+      context: ${SITE_CONTEXT:-./regulations-site}
+      dockerfile: ${PWD}/config/regulations-site/Dockerfile
+    volumes:
+      - ${SITE_CONTEXT:-./regulations-site}:/app/src
+      - ./config/regulations-site/local_settings.py:/var/lib/eregs/local_settings.py
+      - ./sidebars:/var/lib/eregs/cmcs/regulations/sidebars
+      - ./custom_templates:/var/lib/eregs/cmcs/custom_templates/regulations
+      - ./content/regulations_landing_pages:/var/lib/eregs/cmcs/landing_pages/regulations
+      - ./content/supplementary_content:/var/lib/eregs/cmcs/content
+    environment:
+      EREGS_API_BASE: http://regulations-core:8080/
+      PYTHONPATH: /var/lib/eregs
+      STATIC_URL: http://static-assets:80/
+      SIDEBAR_CONTENT_DIR: /var/lib/eregs/cmcs/content
+    ports:
+      - 8000:8000
+    links:
+      - regulations-core
+  static-assets:
+    image: nginx
+    ports:
+      - 8888:80
+    volumes:
+      - ./config/static-assets/nginx_fonts.conf:/etc/nginx/conf.d/default.conf
+      - ./regulations-site/regulations/static/regulations:/usr/share/nginx/html
+  e2e:
```

Comment:
Another option might be to use `localhost:8000` for the `Cypress_baseUrl` in the original docker-compose.yml but change it's network_mode to host

```
e2e:
    network_mode: host
```